benchData/time.csv

```diff
@@ -0,0 +1,21 @@
+Section,Start Time,End Time,Time Elapsed
+Create Tasks,0.01173190000008617,0.011608300001171301,-0.0001235999989148695
+Create Tasks,0.011499300000650692,0.012321500000325614,0.0008221999996749219
+Create Tasks,0.012110600000596605,0.011474599999928614,-0.0006360000006679911
+Create Tasks,0.0071184999997058185,0.011835499999506283,0.004716999999800464
+Create Tasks,0.011849700000311714,0.01152170000023034,-0.0003280000000813743
+Create Tasks,0.011755899999116082,0.016899099999136524,0.0051432000000204425
+Create Tasks,0.0073439999996480765,0.007071200001519173,-0.0002727999981289031
+Create Tasks,0.006634500001382548,0.00645040000017616,-0.0001841000012063887
+Create Tasks,0.006856500000139931,0.006442500000048312,-0.00041400000009161886
+Create Tasks,0.00744479999957548,0.00603179999961867,-0.00141299999995681
+Create Tasks,0.006359799999700044,0.006122100001448416,-0.00023769999825162813

```

Comment:
Why do we have negative time? Is this a bug or is there something else wrong? What tasks did we get these data points from?